docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -115,12 +133,9 @@ Provider's `/config` folder has the same structure of  `/config` folder in CAPI
     ```
     - "--metrics-bind-addr=127.0.0.1:8080"
     ```
-    - Verify that fetaure flags required by your container are properly set
+    - Verify that feature flags required by your container are properly set
       (as it was in `/config/webhook/manager_webhook_patch.yaml`).
-1. Edit the `/config/manager/manager_auth_proxy_patch.yaml` file:
-    - Remove the patch for the container with name `manager`
```

Comment:
I'm not sure we should change those instructions, because I assume some providers already has implemented those changes.

So I think that the changes in this PR should be considered starting from a repository after those change applies (this also should be enforced by the order of changes in this doc + eventually by a note)

Reply:
Could we add some text though that links to the alternative of removing the proxy?

Reply:
Maybe we can phrase it in a way that the kube-rbac proxy removal works for both cases? (something like grep for certain files and remove the files and all references of them)

Reply:
Yeah I was thinking that we could add the instructions for the removal above this change and just say something like `(skip this step if you've done step number X)`